packages/flutter/test/rendering/layers_test.dart

```diff
@@ -239,4 +290,169 @@ void main() {
       layer.shadowColor = const Color(1);
     });
   });
+
+  group('PhysicalModelLayer checks elevations', () {
+    /// Adds the layers to a container where A paints before B.
+    ///
+    /// Expects there to be `errorCount` errors.  Checking elevations is enabled
+    /// by default.
+    void _testConflicts(
+      PhysicalModelLayer layerA,
+      PhysicalModelLayer layerB,
+      int errorCount, [
```

Comment:
I would maybe make the last two parameters named ones with errorCount required (maybe renamed to expectedErrorCoun?). The tests below read a little better that way...

Reply:
Done